docs/en/observability/tab-widgets/uptime-monitoring/run-test-configure-content.asciidoc

```diff
@@ -0,0 +1,133 @@
+:synthetics_version: v1.0.0-beta.26
+
+// tag::managed[]
+
+beta[] Configure each monitor directly in your test code using `monitor.use`.
+You must configure the monitor for each journey, which allows you to set different options for each monitor.
+For example:
+
+[source,js]
+----
+import { journey, step, monitor, expect } from '@elastic/synthetics';
+
+journey('Ensure placeholder is correct', ({ page, params }) => {
+  monitor.use({
+    id: 'example-monitor',
+    schedule: 10,
+    throttling: {
+      download: 10,
+      upload: 5,
+      latency: 100,
+    },
+  });
+  step('Load the demo page', async () => {
+    await page.goto('https://elastic.github.io/synthetics-demo/');
+  });
+  step('Assert placeholder text', async () => {
+    const placeholderValue = await page.getAttribute(
+      'input.new-todo',
+      'placeholder'
+    );
+    expect(placeholderValue).toBe('What needs to be done?');
+  });
+});
+----
+
+Here are all the available configuration options:
+
+`id` (`string`)::
+A unique identifier for this monitor.
+`name` (`string`)::
+A human readable name for the monitor.
+`tags` (`Array<string>`)::
+A list of tags that will be sent with the monitor event. Tags are displayed in the {uptime-app} and allow you to search monitors by tag.
+`schedule` (`number`)::
+At which interval to run the test in seconds.
+`enabled` (`boolean`)::
+Enable or disable the monitor from running without deleting it.
+`locations` (https://github.com/elastic/synthetics/blob/{synthetics_version}/src/dsl/monitor.ts#L36[`Array<SyntheticsLocationsType>`])::
+Where to deploy the monitor. Monitors can be deployed in multiple locations so that you can detect differences in availability and response times across those locations.
+`throttling` (https://github.com/elastic/synthetics/blob/{synthetics_version}/src/common_types.ts#L192-L196[`ThrottlingOptions`])::
+Control the monitor's download speeds, upload speeds, and latency to simulate your application's behavior on slower or laggier networks.
+`screenshot` (https://github.com/elastic/synthetics/blob/{synthetics_version}/src/common_types.ts#L190[`ScreenshotOptions`])::
+Options include `'on'`, `'off'`, or `'only-on-failure'`.
+`params` (https://github.com/elastic/synthetics/blob/{synthetics_version}/src/common_types.ts#L45[`Params`])::
```

Comment:
I would remove these params as they are kind of implicit and will be taken from journey params object.